src/DI/Container.php

```diff
@@ -108,4 +111,17 @@ public function tabs() {
 	public function ui() {
 		return new UIServices($this);
 	}
+
+    /**
+     * Get interface to the http service.
+     *
+     * @return GlobalHttpState
+     */
+	public function http() {
+	    return new HTTPServices(
```

Comment:
Thanks for the quick feedback.

I hope you are right, it would simplify my next task(#467).

That are really good arguments and I fully agree with your opinion. Therefore, I will remove the  method from the Container class to (hopefully) reduce the direct usage of the GlobalHttpState.